CONTRIBUTING.md

```diff
@@ -1 +1,72 @@
-# Contributing
+# Contributing Guidelines
+
+Welcome to the contributing guidelines for the Planet Of The Bugs repository on GitHub! We appreciate your interest in contributing to our project. Whether you're an experienced developer or just starting out, we value your input and contributions.
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+  - [Forking the Repository](#forking-the-repository)
+  - [Setting Up Your Local Environment](#setting-up-your-local-environment)
+- [Contributing](#contributing)
+  - [Creating a New Feature or Bugfix](#creating-a-new-feature-or-bugfix)
+  - [Making Changes](#making-changes)
+  - [Submitting a Pull Request](#submitting-a-pull-request)
+- [Code of Conduct](#code-of-conduct)
+
+## Getting Started
+
+Thank you for considering contributing to our project. Before you begin, please make sure you have a GitHub account. If you don't have one, you can sign up [here](https://github.com/signup).
+
+### Forking the Repository
+
+1. Navigate to the Planet Of The Bugs repository on GitHub: [https://github.com/Emmanuel-Melon/planet-of-the-bugs](https://github.com/Emmanuel-Melon/planet-of-the-bugs)
+2. Click the "Fork" button in the upper right corner of the repository page. This will create a copy of the repository in your GitHub account.
+
+### Setting Up Your Local Environment
+
+1. Clone your forked repository to your local machine: git clone https://github.com/Emmanuel-Melon/planet-of-the-bugs.git
+2. Navigate to the main branch: git checkout dev
+3. Install any necessary dependencies: yarn install && yarn dev
+4. Start the development server by running the following command in your terminal: Navigate to [http://localhost:3000](http://localhost:3000) in your web browser to view the app.
+5. To get started, copy the .example.env file to .env by running the following command in your terminal: cp .example.env .env
```

Comment:
This step should be done before running the app (yarn dev) since running the app without the env variables would lead to errors.